test/polytopes.jl

```diff
@@ -1,5 +1,12 @@
 @testset "Polytopes" begin
   @testset "Segment" begin
+    # constructors
+    s1 = Segment(P2[(1,1),(2,2),(1,1)])
+    s2 = Segment(P2(1,1),P2(2,2),P2(1,1))
+    s3 = Segment(@SVector P2[(1,1),(2,2),(1,1)])
+    s4 = Segment(T.((1,1.)),T.((2.,2.)),T.((1.,1.)))
+    @test s1 == s2 == s3 == s4
```

Comment:
But Segment is only a straight segment between two points. Why we would like to add tests with three points?

Reply:
Ooh. That's a remnant of a copy-paste that I didn't correct. It was originally to test that the constructors we have for polytopes work on something else than a `Chain`.